src/pytorch_lightning/utilities/argparse.py

```diff
@@ -26,6 +26,8 @@
 from pytorch_lightning.utilities.parsing import str_to_bool, str_to_bool_or_int, str_to_bool_or_str
 
 _T = TypeVar("_T", bound=Callable[..., Any])
+FROM_ARGPARSE_ARGS_CLS = Union[Type["ParseArgparserDataType"], Type["pl.LightningDataModule"]]
+FROM_ARGPARSE_ARGS_RETURN = Union["ParseArgparserDataType", "pl.LightningDataModule"]
 
 
 class ParseArgparserDataType(ABC):
```

Comment:
Could this class be removed now?

Reply:
removed in https://github.com/Lightning-AI/lightning/pull/13693/commits/850263e941b63a74355a606664a685aee1c66dcd